packages/icons/scripts/buildSvg.js

```diff
@@ -17,7 +19,7 @@ const normalizeAndCopySvg = (srcDir, distDir) => {
         const svgPath = path.join(srcDir, svg);
         const svgDistPath = path.join(distDir, svg);
         const svgContent = fs.readFileSync(svgPath, 'utf8');
-        const svgContentFixed = svgContent.replace(/fill="#\w+"/g, 'fill="currentColor"');
+        const svgContentFixed = normalizeSvgColors(svg, svgContent);
```

Comment:
I do not like the `svgContentFixed` name. Can we use `svgContentNormalized` instead? Or just append the `replace` behind the call - `const svgSpriteContent = normalizeSvgColors(svg, svgContent).replace(...).replace(...)`?

Reply:
We use it here too
```
fs.writeFileSync(path.join(svgDistPath), svgContentNormalized);
```
But I renamed it to `svgContentNormalized`